src/resources/fundingLogos/mybank.jsx

```diff
@@ -0,0 +1,56 @@
+/* @flow */
+/* @jsx jsxToHTML */
+
+import { jsxToHTML, JsxHTMLNode } from '../../lib/jsx'; // eslint-disable-line no-unused-vars
+import { BUTTON_LOGO_COLOR } from '../../constants';
+
+const MYBANK_LOGO_COLORS = {
+    [ BUTTON_LOGO_COLOR.WHITE ]: {
+        primary:   '#ffffff',
+        secondary: '#ffffff',
```

Comment:
Won't having all paths be the same color create a blank-looking SVG? I think that's what's happening here in [this screenshot](https://github.com/paypal/paypal-checkout/blob/d70d9d19ee3e7dfbb9c288e2e43ad939caf2adb0/test/screenshot/images/button.locale%3Dde_DE%26button.commit%3Dtrue%26button.style.color%3Dblack%26button.style.layout%3Dvertical%26button.style.maxbuttons%3D4%26button.funding.allowed%3Dcredit%2Celv%2Cgiropay%2Csofort.png)

Reply:
Fixed this . please check it now

Reply:
Fixed it now. please check it again